docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    ts = data['T'].reshape(-1,)
</pre><p>Instead of <code>ts</code>, we should be able to call this <code>T</code> now, right?    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Since I am doing <code>T, M_T, Σ_T = core.preprocess(ts, m)</code> later, <code>T</code> is the cleaned version. <strong>It is ok; however,</strong>   since I am comparing  the outputs of merlin with stumpy (for which I used the original time series and not the cleaned version), I keep the variable <code>ts</code> here.</p>

Reply:
Sorry, I forgot to remove this comment once I realized the same thing as you had pointed out. Please ignore